docker/control-host-openstack/README.md

```diff
@@ -29,6 +29,23 @@ There are two options for running the `control-node` containers:
 
 ### Docker Compose (recommended)
 
+#### Environment Variables
+
+The following environment variables can be used to control the execution environment within the control host:
+
+| variable | purpose | default value |
+|:---------|:--------|:--------------|
+|SRC_CODE_DIR|The local source code directory|~/src|
+|DST_CODE_DIR|The remote directory for the source code directory|/root/code|
```

Comment:
Can we say something besides 'remote' here? Destination inside the container? Something about the mount location?